docs/topics/maven.md

```diff
@@ -279,7 +279,7 @@ The following attributes are supported:
 | Name | Property name | Description | Possible values |Default value |
 |------|---------------|-------------|-----------------|--------------|
 | `jvmTarget` | `kotlin.compiler.jvmTarget` | Target version of the generated JVM bytecode | "1.8", "9", "10", ..., "18" | "%defaultJvmTargetVersion%" |
-| `jdkHome` | `kotlin.compiler.jdkHome` |  	Include a custom JDK from the specified location into the classpath instead of the default JAVA_HOME | | |
+| `jdkHome` | `kotlin.compiler.jdkHome` |      Include a custom JDK from the specified location into the classpath instead of the default JAVA_HOME | | |
```

Comment:
```suggestion
| `jdkHome` | `kotlin.compiler.jdkHome` |       Include a custom JDK from the specified location into the classpath instead of the default JAVA_HOME | | |
```